### PR TITLE
Make formula relative to actual directory by using 'tpldir'

### PR DESCRIPTION
**Pull request by mkotsbak**
The rest of the references are impossible to make realtive yes because of https://github.com/saltstack/salt/issues/41195

### CODE REVIEW

**Comment by aboe76** [comment]
@noelmcloughlin can you test this and see if we can merge it?

**Comment by noelmcloughlin** [comment]
Yes, no problem @aboe76 will test today.

File: postgres/macos/postgresapp.sls [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% from "postgres/map.jinja" import postgres as pg with context %}
+{%- from salt.file.dirname(tpldir) ~ "/map.jinja" import postgres with context -%}
```
Comment (noelmcloughlin) [inline postgres/macos/postgresapp.sls]: Value needs to be 'pg'

**Comment by mkotsbak** [comment]
Ah, didn't notice that import line difference. Fixed, rebased and pushed PR branch again.

**Comment by aboe76** [comment]
@mkotsbak and @noelmcloughlin thanks for your hard work

**Comment by aboe76** [comment]
@EvaSDK thanks for the first review

**Pull request merged**